### DISCUSSION TITLE
DRM support for Windows/Xbox platforms

### DISCUSSION THREAD

**Discussion by ChaoyuXiang-Pluto** [TheWidlarzGroup/react-native-video#4960] [Ideas]
### What problem are you trying to solve?

Many streaming services would benefit if react-native-video supported (PlayReady) DRM on Xbox. This would allow them to convert their UWP app on Xbox (mostly web apps) into React Native with a performance gain. Currently both iOS and Android support DRM, leaving only Windows behind.

### Proposed solution

Add DRM support (specifically PlayReady) for the Windows/Xbox platform, bringing it in line with the existing iOS and Android DRM support.

### Platforms

Windows

### Additional context

/cc @namrog84

**Comment by ghost** [comment]
Do you have any ideas in mind? I have a project that requires this, so I thought about trying to implement it :)

**Comment by ProtoScott** [comment]
Is this still a consideration?

**Comment by TWG-AI** [comment]
Thanks for the idea — noted for the team to consider.

> I'm an AI assistant, so double-check any specifics I may have mentioned.
